src/components/cards/controltable/ResponsiveRowLarge.vue

```diff
@@ -22,7 +22,7 @@
       </v-card>
     </v-col>
 
-    <v-col class="pa-0" :cols="2">
+    <v-col class="pa-0" :cols="3">
       <v-card tile flat class="fill-height">
         <slot name="tags" />
```

Comment:
This slot needs to be updated to change the column title to : `NIST SP 800-53 Security Controls & CCIs`